src/patternfly/upgrade-examples/PopoverUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.popovers-pf` | `<div>` | `pf-c-popover` | `<div>` | Ceates parent container that holds the popover. |
+| `.container-popovers-pf` | `<div>` | `.pf-c-popover__content` | `<div>` | Creates the container for the popover itself. |
+| `.top` | `<div>` | `.pf-m-top` | `<div>` | Directs the direction of the popover arrow. Required. |
+| `.right` | `<div>` | `.pf-m-right` | `<div>` | Directs the direction of the popover arrow. Required. |
+| `.bottom` | `<div>` | `.pf-m-bottom` | `<div>` | Directs the direction of the popover arrow. Required. |
+| `.left` | `<div>` | `.pf-m-left` | `<div>` | Directs the direction of the popover arrow. Required. |
+| `.arrow` | `<div>` | `.pf-c-popover__arrow` | `<div>` | Creates an arrow pointing towards the element the popover describes. Required. |
+| `.close` | `<span>` |`.pf-c-popover__close` | `<div>` | Creates the closing button for the popover. |
+| `.popover-title` | `<h3>` | `.pf-c-popover__header-title` | `<h1>` | Creates popover title. |    
```

Comment:
add in this line, it doesnt matter that theres no direct comparison in pf3, because this class is necessary for pf4.

| | | `.pf-c-popover__header` | `<header>` | Creates Popover Header. |

Reply:
"| -- | -- | `.pf-c-popover__header` | `<header>` | Creates Popover Header. |" has been added to `code.md` file.